Templates/DefaultProject/Template/project.json

```diff
@@ -1,5 +1,6 @@
 {
     "project_name": "${Name}",
+    "version": "1.0.0",
```

Comment:
What is the rationale for the Projects, not have support for a a ${version} placeholder?

Reply:
Good catch!  I forgot to update these.